src/util/voxel_io.cpp

```diff
@@ -69,22 +71,29 @@ from_qb(
     std::array<int32_t, 3>& center, std::array<uint32_t, 3>& size
 )
 {
-    static quill::Logger* logger = logging::get_logger("voxel_io");
+    // static quill::Logger* logger = logging::get_logger("voxel_io");
 
-    LOG_INFO(logger, "Reading voxels from {}", path);
+    LOG_INFO(
+        logging::file_io_logger, "Reading voxels from {}.",
+        path.lexically_normal().string()
```

Comment:
Already garunteed to be lexically normal in `utils/files.hpp`

Also don't need the cast to string.

Reply:
Casting to string means that I can follow the link in the log file with vs code. if not there are double slashes, and quotes